baselines/FedMLB/README.md

```diff
@@ -0,0 +1,588 @@
+  </tr>
+</table>
+
+![cifar100, 100 clients, 5% participation rate, alpha = 0.3](_static/accuracy_comparison_cifar100_dirichlet_0.3_100_.png)
+
+#### Tiny-ImageNet, Dir(0.3), 100 clients, 5% participation.
+
+<table>
+  <tr>
+    <td></td>
+    <td colspan="2">Accuracy @500R</td>
+    <td colspan="2">Accuracy @1000R</td>
+  </tr>
+  <tr>
+    <td>Method</td>
+    <td>Paper</td>
+    <td>This repo</td>
+    <td>Paper</td>
+    <td>This repo</td>
+  </tr>
+  <tr>
+    <td>FedAvg</td>
+    <td>33.94</td>
+    <td>33.39</td>
+    <td>35.42</td>
+    <td>35.78</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td>
+    <td>37.20</td>
+    <td>35.42</td>
+    <td>40.16</td>
+    <td>39.14</td>
+  </tr>
+</table>
+
+![tiny imagenet, 100 clients, 5% participation rate, alpha = 0.3](_static/accuracy_comparison_tiny-imagenet_dirichlet_0.3_100_.png)
+
+
+### Table 1b
+
+The results of Table 1a in the paper (for FedAvg and FedMLB) refers
+to CIFAR-100 (Figure 3) and Tiny-ImageNet (Figure 7b) with 
+the setting (2.) Large-scale experiments with Dir(0.3), 500 clients, 
+2% participation rate.
+
+To reproduce the results run the following:
+```bash
+python -m FedMLB.main --multirun dataset_config.dataset="cifar100","tiny-imagenet" algorithm="FedMLB","FedAvg","FedAvg+KD" total_clients=500 clients_per_round=10
+```
+#### CIFAR-100, Dir(0.3), 500 clients, 2% participation.
+
+<table>
+  <tr>
+    <td></td>
+    <td colspan="2">Accuracy @500R</td>
+    <td colspan="2">Accuracy @1000R</td>
+  </tr>
+  <tr>
+    <td>Method</td>
+    <td>Paper</td>
+    <td>This repo</td>
+    <td>Paper</td>
+    <td>This repo</td>
+  </tr>
+  <tr>
+    <td>FedAvg</td>
+    <td>29.87</td>
+    <td>32.54</td>
+    <td>37.48</td>
+    <td>38.75</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td>
+    <td>32.03</td>
+    <td>36.63</td>
+    <td>42.61</td>
+    <td>46.65</td>
+  </tr>
+</table>
+
+![cifar100, 500 clients, 2% participation rate, alpha = 0.3](_static/accuracy_comparison_cifar100_dirichlet_0.3_500_.png)
+
+#### Tiny-ImageNet, Dir(0.3), 500 clients, 2% participation.
+
+
+<table>
+  <tr>
+    <td></td>
+    <td colspan="2">Accuracy @500R</td>
+    <td colspan="2">Accuracy @1000R</td>
+  </tr>
+  <tr>
+    <td>Method</td>
+    <td>Paper</td>
+    <td>This repo</td>
+    <td>Paper</td>
+    <td>This repo</td>
+  </tr>
+  <tr>
+    <td>FedAvg</td>
+    <td>23.63</td>
+    <td>22.62</td>
+    <td>29.48</td>
+    <td>28.37</td>
+  </tr>
+  <tr>
+    <td>FedMLB</td>
+    <td>28.39</td>
+    <td>26.84</td>
+    <td>33.67</td>
+    <td>33.13</td>
+  </tr>
+</table>
+
+![tiny imagenet, 500 clients, 2% participation rate, alpha = 0.3](_static/accuracy_comparison_tiny-imagenet_dirichlet_0.3_500_.png)
+
+### Table 3
+
+To reproduce results reported in Table 3 of the paper,
+resulting from _more local iterations_ (K=100 or K=200 in the
+paper, instead of K=50), run the following:
+```bash
+python -m FedMLB.main --multirun algorithm="FedMLB","FedAvg","FedAvg+KD" local_updates=100 # or local_updates=200 
```

Comment:
If you have the results of this experiment, could you add them to a table? (just like you did for the Table 1a and 1b)

Reply:
Currently collecting results.